javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -9,11 +9,14 @@ import NavbarHover from "../Nav/NavbarHover";
 
 export function PureNavbar({ isAdmin, isMember, channelPages, adminPages, searchPages, dataPages }) {
   return (
-    <Navbar bg="dark" variant="dark" expand="lg">
+    <Navbar className="navbar-custom" expand="lg" bg="dark" variant="dark">
       <Navbar.Toggle />
       <Navbar.Collapse>
         <LinkContainer to="">
-          <Navbar.Brand data-testid="brand">Mapache Search</Navbar.Brand>
+        <Navbar.Brand data-testid="brand">
+          <p className="brand"><img className="brand" src={'navbarlogo.png'}  alt="Mapache Search icon"  /></p>
```

Comment:
The logo only shows up on the main page, and does not show up on other pages. Please fix this such that it will show up on every page.